datasets/hebrew_this_world/README.md

```diff
@@ -0,0 +1,181 @@
+- expert-generated
+language_creators:
+- found
+languages:
+- he
+licenses:
+- gpl
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- sequence-modeling
+task_ids:
+- language-modeling
+---
+
+# Dataset Card for HebrewSentiment
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://thisworld.online/
+- **Repository:** https://github.com/thisworld1/thisworld.online
+- **Paper:** 
+- **Leaderboard:**
+- **Point of Contact:** 
+
+### Dataset Summary
+
+HebrewThisWorld is a data set consists of 2028 issues of the newspaper 'This World' edited by Uri Avnery and were published between 1950 and 1989. Released under the AGPLv3 license.
+
+Data Annotation: 
+
+### Supported Tasks and Leaderboards
+
+Language modeling
+
+### Languages
+
+Hebrew
+
+## Dataset Structure
+
+csv file with "," delimeter
+
+### Data Instances
+
+Sample:
+
+| issue\_num | page\_count | date       | date\_he        | year | href                               | pdf                                               | coverpage                                         | backpage                                          | content                                               | url                                    | 
+| ---------- | ----------- | ---------- | --------------- | ---- | ---------------------------------- | ------------------------------------------------- | ------------------------------------------------- | ------------------------------------------------- | ----------------------------------------------------- | -------------------------------------- | 
+| 2403       | 72          | 1983-09-20 | 20 בספטמבר 1983 | 1983 | https://thisworld.online/1983/2403 | https://olam.eu-central-1.linodeobjects.com/pd... | https://olam.eu-central-1.linodeobjects.com/pa... | https://olam.eu-central-1.linodeobjects.com/pa... | ,י״ג תשרי תשמ״ד20.9.83 ,\\n\\nמיסטר 2403\\n\\nכתבת... | https://thisworld.online/api/1983/2403 |
```

Comment:
Nice! Would you mind formatting the data instance in JSON format?

Reply:
agree, fixed